### PR TITLE
[stable8] backport PHPDoc fixes

### PR DESCRIPTION
**Pull request by MorrisJobke**
Backport  #16985 and #16995
cc @nickvergessen @lukasreschke @karlitschek

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @DeepDiver1975

File: lib/public/iservercontainer.php [hunk 1]
```diff
@@ -251,11 +251,11 @@ function getSearch();
...
 	 * Get the certificate manager for the user
 	 *
-	 * @param \OCP\IUser $user (optional) if not specified the current loggedin user is used
-	 * @return \OCP\ICertificateManager
+	 * @param string $userId (optional) if not specified the current loggedin user is used
+	 * @return \OCP\ICertificateManager | null if $userId is null and no user is logged in
 	 */
-	function getCertificateManager($user = null);
-	
+	function getCertificateManager($userId = null);
```
Comment (nickvergessen) [inline lib/public/iservercontainer.php]: uid like on the class?

Comment (DeepDiver1975) [inline lib/public/iservercontainer.php]: I prefer userId in both

Comment (MorrisJobke) [inline lib/public/iservercontainer.php]: But not in this backport. I can do this on master, but not in a backport in already reviewed PRs for master ;)

Comment (nickvergessen) [inline lib/public/iservercontainer.php]: Also missing visibility, so yes, please fix on master...

**Comment by nickvergessen** [comment]
:+1: fine by me

the remaining stuff should be addressed in master

File: lib/public/iservercontainer.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/public/iservercontainer.php]: > Also missing visibility, so yes, please fix on master...

Which was fixed on master. -> no fix in backport

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**